### ISSUE TITLE
Fatal error C1083: Cannot open include file: 'xercesc/dom/DOMDocument.hpp': No such file or directory

### ISSUE THREAD

**Issue by buzz3791** [MicrosoftDocs/cpp-docs#2320]
I'm encountering this problem when this code
#include <xercesc/dom/DOMDocument.hpp>
should match this cl.exe option:
/I"C:\mv76repo\NativeThirdParty\xerces-c\xerces-c-2.8.0-mv1-vs2019-win32-asan\src"

which should result in the compiler successfully picking:
C:\mv76repo\NativeThirdParty\xerces-c\xerces-c-2.8.0-mv1-vs2019-win32-asan\src\xercesc\dom\DOMDocument.hpp

But this instead causes:
1>C:\dev\mv76\PersistenceService\libPersistenceService\src\BrokerCheckpointImpl.h(7,10): fatal error C1083: Cannot open include file: 'xercesc/dom/DOMDocument.hpp': No such file or directory

This problem goes away if I change the .hpp's path to:
C:\mv76repo\NativeThirdParty\xercesc\src\xercesc\dom\DOMDocument.hpp

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d846b08f-ccc3-d619-d710-3a526e00d5bc
* Version Independent ID: 11f14d2a-affb-9e33-6e76-4983389e52b1
* Content: [Fatal Error C1083](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/fatal-error-c1083?view=vs-2019)
* Content Source: [docs/error-messages/compiler-errors-1/fatal-error-c1083.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/error-messages/compiler-errors-1/fatal-error-c1083.md)
* Product: **visual-cpp**
* Technology: **cpp-diagnostics**
* GitHub Login: @corob-msft
* Microsoft Alias: **corob**

**Comment by corob-msft** [comment]
@buzz3791 
I'm sorry you're having trouble. We don't deal with product support issues here, and this doesn't seem like an issue with the C1083 documentation, though it might be. As far as I can tell, what you've described should work if the specified file exists at that location. If the compiler is having trouble with a particular element in the path, such as *`xerces-c\xerces-c-2.8.0-mv1-vs2019-win32-asan`* then it may be a bug. I'd try the Visual Studio C++ [Developer Community](https://developercommunity.visualstudio.com/spaces/62/index.html) website for issues like this. Best of luck.

**Comment by buzz3791** [comment]
Seems like it was a bug.  After several hours, I finally got Visual Studio C++ "unstuck" by disabling:
"Project Properties->C/C++->Additional Include Directories->dropdown->Edit ...->Checkbox Inherit from parent or project defaults"
This seems to have triggered some kind of rebuild of a cache of valid header files or something.  Now the compiler error has disappeared.  I went back into the same dialog and re-enabled "Inherit from parent or project defaults" and things are fine.

Thanks, I'll report the bug.
